patches/chrome-browser-ui-views-tabs-tab_style.cc.patch

```diff
@@ -37,11 +28,14 @@ index 8f21aa7edbb3b0409c68e0d850541573710f993c..791e5dc1c2c161325990a43f0bb2d960
                   separator_size.width(), separator_size.height());
  
    separator_bounds.trailing = separator_bounds.leading;
-@@ -696,12 +697,13 @@ void GM2TabStyle::PaintSeparators(gfx::Canvas* canvas) const {
+@@ -696,12 +697,16 @@ void GM2TabStyle::PaintSeparators(gfx::Canvas* canvas) const {
                                                     SK_AlphaOPAQUE));
    };
  
-+  const int separator_radius = separator_bounds.leading.width() / 2;
++  // Even if |separator_radius| becomes 1 native pixel the 'roundedness'
```

Comment:
we should avoid comments in patches. Any info should be in the commit message or PR and can be found with git blame

Reply:
I'm ok either way but just noting that I don't think it costs extra time to rebase because of it.